src/main/java/com/google/sps/data/PathFinder.java

```diff
@@ -0,0 +1,23 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+import com.google.maps.errors.ApiException;
+import java.io.IOException;
+import java.lang.InterruptedException;
+
+interface PathFinder {
+  public int getTimeBetweenPoints(Point a, Point b)  throws ApiException, IOException, InterruptedException, DataNotFoundException ;
```

Comment:
Right, but Directions API returns the time needed to get from one point to another as an integer representing the number of seconds. I'm not sure how to represent it using LocalTime since `getTimeBetweenPoints` doesn't depend on time.
Is it okay to rename the method instead? (e.g. `getNumberOfSecondsBetweenPoints` )